tox.ini

```diff
@@ -14,10 +14,11 @@ setenv =
 deps =
 	coverage
 	dj-database-url
-	django110: Django >= 1.10, < 1.11
 	django111: Django >= 1.11, < 2.0
+	django200: Django ~= 2.0b1
 	djangomaster: https://github.com/django/django/archive/master.tar.gz
 	postgres: psycopg2
+pip_pre = True
```

Comment:
This will not be required once django 2.0 comes out. Also, the line above will need to change to `Django ~= 2.0.0`

Reply:
This can be updated to Django 2.0rc1

Reply:
We can now:

- remove ``pip_pre`` line
- ``django200: Django >= 2.0, < 2.1``